airflow/cli/commands/db_command.py

```diff
@@ -136,21 +144,30 @@ def migratedb(args):
         print(f"Performing upgrade to the metadata database {settings.engine.url!r}")
     else:
         print("Generating sql for upgrade -- upgrade commands will *not* be submitted.")
-
-    db.upgradedb(
-        to_revision=to_revision,
-        from_revision=from_revision,
-        show_sql_only=args.show_sql_only,
-        reserialize_dags=args.reserialize_dags,
-    )
+    if reserialize_dags:
+        command(
+            to_revision=to_revision,
+            from_revision=from_revision,
+            show_sql_only=args.show_sql_only,
+            reserialize_dags=True,
+        )
+    else:
+        command(
+            to_revision=to_revision,
+            from_revision=from_revision,
+            show_sql_only=args.show_sql_only,
+        )
```

Comment:
```suggestion
    command(
        to_revision=to_revision,
        from_revision=from_revision,
        show_sql_only=args.show_sql_only,
        reserialize_dags=reserialize_dags,
    )
```

Reply:
The FAB side of the command doesn't have the option to reserialize_dags after upgrade, and I think it shouldn't. We should reserve dag reserialization to when the airflow db migrate is run. I am happy to change it if we want this on the external db commands too, but I don't see much point to it.

Reply:
Ah, sorry, fair enough!